### PR TITLE
refactor: minor changes

### PR DESCRIPTION
**Pull request by jotatoledo**

### CODE REVIEW

File: src/app/operators/operators.component.html [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
+               [fixedTopGap]="operatorMenuGap$ | async"
+               fixedInViewport
                class="operator-list-sidenav"
                #operatorSidenav>
     <mat-nav-list *ngFor="let category of categories"
                   class="operator-list">
       <h3 mat-subheader
           class="category-subheader">{{ category }}</h3>
       <a mat-list-item
+         (click)="smallScreen && operatorSidenav.close()"
```
Comment (jsonberry) [inline src/app/operators/operators.component.html]: 👍

Comment (jotatoledo) [reply src/app/operators/operators.component.html]: Ill do this in a separate PR

File: src/app/operators/operators.component.ts [hunk 2]
```diff
@@ -61,18 +63,38 @@ interface OperatorDocMap {
...
-  @ViewChild(MatSidenav) _sidenav: MatSidenav;
-  public groupedOperators: OperatorDocMap;
-  public categories: string[];
   private _onDestroy = new Subject();
 
+  @ViewChild(MatSidenav) _sidenav: MatSidenav;
+  groupedOperators: OperatorDocMap;
+  categories: string[];
+  smallScreen$: Observable<boolean>;
+  operatorMenuGap$: Observable<number>;
```
Comment (jsonberry) [inline src/app/operators/operators.component.ts]: Technically speaking, I know these will be declared as public class members, but that's not the style I've seen in the project thus far.

File: src/app/operators/operators.component.ts [hunk 3]
```diff
@@ -61,18 +63,38 @@ interface OperatorDocMap {
...
   constructor(
     private _breakpointObserver: BreakpointObserver,
     private _operatorMenuService: OperatorMenuService,
     @Inject(OPERATORS_TOKEN) public operators: OperatorDoc[]
   ) {}
 
   ngOnInit() {
+    this.smallScreen$ = this._breakpointObserver
+      .observe('(max-width: 901px)')
+      .pipe(map(breakPointState => breakPointState.matches));
```
Comment (jsonberry) [inline src/app/operators/operators.component.ts]: 👍

File: src/app/operators/operators.component.ts [hunk 4]
```diff
@@ -84,24 +106,10 @@ export class OperatorsComponent implements OnInit, AfterContentInit, OnDestroy {
       .subscribe(_ => this._sidenav.open());
   }
 
-  get extraSmallScreen() {
-    return this._breakpointObserver.isMatched('(max-width: 601px)');
-  }
-
-  get smallScreen() {
+  get smallScreen(): boolean {
```
Comment (jsonberry) [inline src/app/operators/operators.component.ts]: Remove this, now that you'd have `this.smallScreen$`?

**Comment by jotatoledo** [comment]
Ill redo, as I didnt respected the contributing guidelines :smile:

**Pull request closed without merging**